app/views/connectors/after_post_body/comments.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="qa-comments">
+  <% post.question_answer_comments.each_with_index do |comment, i| %>
+    <div class="qa-comment-row">
+      <span class="qa-comment-cooked" itemprop="comment"><%= comment.cooked.html_safe %></span>
+      <% if (u = comment.user) %>
```

Comment:
Just curious, can a comment have no user?

Reply:
No, looking at the model comment can't be created without a user record.

https://github.com/discourse/discourse-question-answer/blob/main/app/models/question_answer_comment.rb#L67

Reply:
It seems like this is still unresolved.

Reply:
Fixed.